src/jobsautoreport/main.py

```diff
@@ -0,0 +1,44 @@
+from datetime import datetime, timedelta
+
+from opensearchpy import OpenSearch
+from slack_sdk.webhook import WebhookClient
+
+from jobsautoreport import config
+from jobsautoreport.process import Processor
+from jobsautoreport.query import Querier
+
+
+def main() -> None:
+
+    os_usr = config.ES_USER
+    os_pwd = config.ES_PASSWORD
+    os_host = config.ES_URL
+
+    client = OpenSearch(os_host, http_auth=(os_usr, os_pwd))
+
+    index_name = config.ES_JOB_INDEX
+
+    data = {}
+
+    now = datetime.now()
+    a_week_ago = now - timedelta(weeks=1)
+
+    querier = Querier(client, index_name)
+    processor = Processor(querier)
+
+    success_rate = processor.get_success_rate(a_week_ago, now)
+    data["success_rate"] = str(success_rate)
+
+    number_of_jobs_triggered = processor.get_number_of_jobs_triggered(a_week_ago, now)
+    data["jobs_triggered"] = str(number_of_jobs_triggered)
+
+    average_job_duration = processor.get_average_duration_of_jobs(a_week_ago, now)
+    data["average_job_duration"] = str(average_job_duration)
```

Comment:
As an improvement for later, it could be nice for the processor to return a unique data structure that contains all the data aggregated.